config.toml

```diff
@@ -33,35 +33,11 @@ theme = "canvas"
       weight = 2
       parent="documentation"
   [[menu.main]]
-      identifier = "doc_installation"
-      name = "Installation"
-      url = "/documentation/installation"
+      identifier = "doc_gettingstarted"
+      name = "Getting Started"
+      url = "/documentation/getting-started"
```

Comment:
don't change this - we have "https://kiali.io/gettingstarted" URL all over external - tweets, upstream Istio docs, etc.

Reply:
OK, that's a bit of a pain the way things are structured, but maybe I'll see what I can do to preserve that link.

Reply:
I found the way to alias previous urls so this should be fine.